CONTRIBUTING.md

```diff
@@ -17,6 +17,51 @@ limitations under the License.
 
 # Contribution Guidelines
 
+## External Contributors: Issue-First Workflow
+
+### When Is a GitHub Issue Required?
+
+**You can submit a PR directly without an issue if:**
+
+- Your change is **<100 lines of code** AND addresses a simple, focused concern (typos, simple bug fixes, formatting)
```

Comment:
I think this is still a loophole that could be abused. I am fine starting with this, but we may need to be prepared to go with no exceptions.  For example, I could make a small <100 line code change that has a very huge impact (i.e. API behavior or adding an extra parameter to an API). 

But I guess you cover this with the *AND* statement here.